src/tests/integration/hrrr-total-precip/test_hrrr_total_precip_20220818_f06.py

```diff
@@ -1,78 +1,65 @@
 import os
-import unittest
 from osgeo import gdal
 from urllib.request import urlretrieve
 
 from cumulus_geoproc.processors import geo_proc
 from cumulus_geoproc.utils.cgdal import find_band
 
 
-class TestHrrrTotalPrecip20220818f06(unittest.TestCase):
+class TestHrrrTotalPrecip20220818f06:
     """HRRR Total Precip; File from 2022-08-18; Forecast Hour 06"""
 
-    def setUp(self) -> None:
+    def setup_method(self) -> None:
         self.acquirable = os.path.join(
-            os.path.dirname(__file__), "..", "fixtures", "hrrr-total-precip", "hrrr-total-precip-20220818", "hrrr.t00z.wrfsfcf06.grib2"
+            os.path.dirname(__file__),
+            "..",
+            "fixtures",
+            "hrrr-total-precip",
+            "hrrr-total-precip-20220818",
+            "hrrr.t00z.wrfsfcf06.grib2",
         )
         # Create parent directory to download acquirable if it dos not exist
         os.makedirs(os.path.dirname(self.acquirable), exist_ok=True)
         # Output Directory
-        self.output_directory = '/output/hrrr-total-precip/hrrr-total-precip-20220818'
+        self.output_directory = "/output/hrrr-total-precip/hrrr-total-precip-20220818"
         os.makedirs(self.output_directory, exist_ok=True)
         # If acquirable not stored in `cumulus-geoproc-test-data`; download it
         if not os.path.isfile(self.acquirable):
             # @todo; change link below to persistent archive link to acquire file
             urlretrieve(
-                'https://noaa-hrrr-bdp-pds.s3.amazonaws.com/hrrr.20220817/conus/hrrr.t00z.wrfsfcf06.grib2',
-                self.acquirable
+                "https://noaa-hrrr-bdp-pds.s3.amazonaws.com/hrrr.20220818/conus/hrrr.t00z.wrfsfcf06.grib2",
+                self.acquirable,
             )
 
-
-    def tearDown(self) -> None:
+    def teardown_method(self) -> None:
         pass
 
     def test_input_file_exists(self) -> None:
-        self.assertTrue(os.path.isfile(self.acquirable))
-    
+        assert os.path.isfile(self.acquirable), "Input file not found"
+
     def test_find_correct_band(self) -> None:
         # Get Band using attributes from processor
         attr = {
-            "GRIB_ELEMENT": "APCP",
+            "GRIB_ELEMENT": "APCP01",
             "GRIB_COMMENT": "precipitation",
             "GRIB_UNIT": "[kg/(m^2)]",
         }
-        # @todo: these are the attributes (below) that should be used to find the 1-hr precipitation band we want
-        #        the above parameters are currently used in geoprocess, causing the wrong band to be selected.
-        # attr = {
-        #     "GRIB_ELEMENT": "APCP01",
-        #     "GRIB_COMMENT": "precipitation",
-        #     "GRIB_UNIT": "[kg/(m^2)]",
-        # }
-        ds = gdal.Open(self.acquirable)
-        band = find_band(ds, attr)
-        ds = None
-        # Get the band; 90 is the band we want in the sample file
-        self.assertEqual(band, 90)
 
+        ds = gdal.Open(self.acquirable)
```

Comment:
Recommend to avoid memory leak:
```
band = find_band(ds, attr)
ds = None

assert band == 90, "Incorrect band number."
```

Reply:
Will change, but memory leaks in ephemeral testing container is not an issue.

Reply:
Agree, unless the test suite gets really long.

Reply:
Thanks for changing - resolving now.